tool/teleport/common/sftp.go

```diff
@@ -651,6 +688,25 @@ func onSFTP() error {
 		serveErr = trace.Wrap(serveErr)
 	}
 
+	// Send a summary event last
+	summaryEvent := &apievents.SFTPSummary{
```

Comment:
What happens if there's an error before the transfer completes? Will we not audit in that case?

Reply:
A few lines above I just set the error to `serveErr` and send the summary event no matter what. Then I return `serveErr` so this event should always be emitted save for a panic or something